GiEnJul/Entities/Giver.cs

```diff
@@ -10,21 +10,15 @@ public class Giver : EntityBase
         public Giver()
         { }
 
-        public Giver(string location, string rowKey)
+        public Giver(string location, string eventName) : base($"{eventName}_{location}", Guid.NewGuid().ToString())
```

Comment:
Could it be an idea to have fks `eloc_{eventName}_{locaction}` as primary key? The `eloc` part would be nice to have if there some time in the future would be added another type of partition key (fks `loc_{location}`).

Reply:
I think this would add more complexity than needed. Givers could be fetched with EventName or Location utilizing the properties: 
![image](https://user-images.githubusercontent.com/45152025/124257776-fa574780-db2c-11eb-99ad-31609d6b7f21.png)